mephisto/tools/scripts.py

```diff
@@ -239,6 +239,7 @@ def build_custom_bundle(custom_src_dir):
     build_path = os.path.join(prebuild_path, "build", "bundle.js")
 
     # see if we need to rebuild
```

Comment:
The code below is an optimization that is commented out. 

**I probably need some feedback if this is the correct way to go.** 

What this code does is that it forgoes the `npm build` and `npm link` commands when it detects that nothing about the webapp changes. However, switching from 

```bash
python run_task.py
```

to 

```bash
python run_task.py mephisto.task.package_to_link_to=mephisto-task
```
requires the npm link command to gets executed. As nothing changed about the webapp the optimization is ran and the npm link command is skipped.

The optimization is pretty detrimental to local package development as you are forced to do quirky things like adding random text to the webapp to force the build and link. This is really unintuitive and personally has made me waste a lot of time in trying to fix.

The optimization itself only really seems to save 1-2 seconds for each time python run_task.py is ran.

Outside of commenting it out, another solution could be to create a new command line arg named, "always_build", which would skip the optimization when set to true. 

Thoughts 🤔?